core/domain/activity_services.py

```diff
@@ -83,8 +117,17 @@ def split_by_type(activity_references):
     The elements in each of the returned lists are in the same order as those
     in the input list.
 
-    If any activity reference is incorrectly formatted, this raises an
-    Exception.
+    Args:
+        activity_references: A list containing activity references.
+    
+    Returns:
+        A list containing exploration ID and the second containing 
```

Comment:
A 2-tuple. The first element is a list of all exploration ids represented in the input list, and the second element is a list of all collection ids represented in the input list.